pythonx/UltiSnips/snippet_manager.py

```diff
@@ -197,6 +197,24 @@ def expand_or_jump(self):
             vim_helper.command("let g:ulti_expand_or_jump_res = 0")
             self._handle_failure(self.expand_trigger, True)
 
+    @err_to_scratch_buffer.wrap
+    def jump_or_expand(self):
+        """This function is used for people who wants to have the same trigger
```

Comment:
nit: people who want

Reply:
Fixed. It's "people who wants" in `expand_or_jump` function and I fix it too.